docs/sources/datasources/prometheus/configure/aws-authentication.md

```diff
@@ -0,0 +1,277 @@
+
+```yaml
+# Example provisioning configuration
+apiVersion: 1
+datasources:
+  - name: Amazon Managed Prometheus
+    type: prometheus
+    access: proxy
+    url: https://aps-workspaces.us-west-2.amazonaws.com/workspaces/ws-12345678-1234-1234-1234-123456789012/
+    jsonData:
+      sigV4Auth: true
+      sigV4AuthType: default
+      sigV4Region: us-west-2
+```
+
+# Migrate to Amazon managed service for Prometheus
+
+Learn more about why this is happening: [Prometheus data source update: Redefining our big tent philosophy](https://grafana.com/blog/2025/06/16/prometheus-data-source-update-redefining-our-big-tent-philosophy/)
+
+## Before you begin
+
+- Ensure you have the organization administrator role
+- If you are self hosting Grafana, back up your existing dashboard configurations and queries
+
+## Migrate on Grafana cloud
+
+Grafana cloud users will be automatically migrated to the relevant version of Prometheus. No action needs to be taken.
+
+## Migrate in air-gapped environments
+
+For air-gapped Grafana installations:
+
+1. Download and install [Amazon Managed Service for Prometheus](https://grafana.com/grafana/plugins/grafana-amazonprometheus-datasource/)
+2. Follow the standard migration process
+
+## Migrate
+
+1. Enable the `prometheusTypeMigration` feature toggle. For more information on feature toggles, refer to [Manage feature toggles](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/#manage-feature-toggles). 
+2. Restart Grafana for the changes to take effect.
+
+**Note**: This feature toggle will be removed in Grafana 13, and the migration will be automatic.
+
+## Check migration status
+
+To determine if your Prometheus data sources have been migrated:
+
+1. Navigate to **Connections** > **Data sources**
+2. Select your Prometheus data source
+3. Look for a migration banner at the top of the configuration page
+
+The banner displays one of the following messages:
+
+- **"Migration Notice"** - The data source has already been migrated
+- **"Deprecation Notice"** - The data source has not been migrated
+- **No banner** - No migration is needed for this data source
+
+## Troubleshooting
+
+### Common migration issues
+
+**Migration banner not appearing**
+- Verify the `prometheusTypeMigration` feature toggle is enabled
+- Restart Grafana after enabling the feature toggle
+
+**Amazon managed service for Prometheus is not installed**
+- Verify that Amazon managed service for Prometheus is installed by going to **Connections** > **Add new connection** and search for "Amazon managed service for Prometheus"
+- Install Amazon managed service for Prometheus if not already installed
+
+### Rolling the migration back without a backup
+
+If you do not have a backup of your Grafana before the migration, you can run the script below. It will find all the Amazon managed service for Prometheus data source instances that were migrated from core Prometheus and revert them back to core Prometheus.
```

Comment:
```suggestion
If you do not have a backup of your Grafana instance before the migration, you can run the script below. It will find all the Amazon Managed service for Prometheus data source instances that were migrated from core Prometheus and revert them back to core Prometheus.
```